gnocchi/carbonara.py

```diff
@@ -148,7 +148,16 @@ def max(self):
         return make_timeseries(self.tstamps, values)
 
     def median(self):
-        return self._scipy_aggregate(ndimage.median)
+        ordered = self._ts['values'].argsort()
+        uniq_inv = numpy.repeat(numpy.arange(self.counts.size), self.counts)
+        max_pos = numpy.zeros(self.tstamps.size, dtype=numpy.int)
+        max_pos[uniq_inv[ordered]] = numpy.arange(self._ts.size)
+        mid_diff, odd = numpy.divmod(self.counts, 2)
```

Comment:
Can we use "floor_divide(self.count, 2), reminder(self.count, 2)" for numpy < 1.13 and avoid to raise the requirement ? Centos/RHEL RDO ships only numpy 1.11 for queens

Reply:
kk. i'll add it as a todo.

Reply:
hm we can't add this as a todo if we don't want to depends on 1.9.0 and not 1.13.0, no?

Reply:
NOTE then? i'm not sure what you mean to be honest.

Reply:
@chungg Sorry I misunderstood you. I thought you were only gonna add a TODO and not change the code at all lol. That's all good, thanks!

Reply:
ah. i see :) i am that lazy so it was a good assumption.lol